src/index.tsx

```diff
@@ -186,6 +188,36 @@ export function generateTrigger(
 
     componentDidMount() {
       this.componentDidUpdate();
+      // This is a special case for handling disabled elements. When a tooltip is
+      // shown on a tooltip element, the flag is activated and the next time this
+      // event listener is called the tooltip is hidden. This happens because
+      // a mouseover event will be triggered when the cursor leaves a disabled
+      // element.
+      // See: https://github.com/facebook/react/issues/4251#issuecomment-334266778
+      document.addEventListener('mouseover', e => {
```

Comment:
Is there a way to save this? Instead of creating event listener in every trigger.

Reply:
@zombieJ you mean extracting the eventListener function into a class variable?

Reply:
Maybe create a observer util that with only one event listener with document.